src/loaders/TiffLoader.ts

```diff
@@ -121,35 +124,49 @@ function getOMEDims(imageEl: Element): OMEDims {
 }
 
 const getBytesPerSample = (type: string): number => (type === "uint8" ? 1 : type === "uint16" ? 2 : 4);
+const getPixelType = (pxSize: number): string => (pxSize === 1 ? "uint8" : pxSize === 2 ? "uint16" : "uint32");
 
 // Despite the class `TiffLoader` extends, this loader is not threadable, since geotiff internally uses features that
 // aren't available on workers. It uses its own specialized workers anyways.
 class TiffLoader extends ThreadableVolumeLoader {
-  url: string;
+  url: string[];
```

Comment:
if in this case the url is always an array of urls can you call this `scenes` or `urls`? or is this trying to be consistent with the parent class?